navBar links (not complete)

melissagio opened the pull request:
incomplete, got the links working, need to fix styling

melissagio committed:
navBar links (not complete)

incomplete, got the links working, need to fix styling

rane20 reviewed (commented):
Looks good so far, just add the navbar.jsx file into a components folder under src. so basically the path should be src/components/NavBar.jsx 
Thanks!

melissagio committed:
melissa_navBar

just need to add drop down error next to about and slightly make logo bigger

melissagio committed:
styling of links

melissagio committed:
home tab in navBar & WIT logo

suggested changes 11.25 - home tab and using full WIT logo

currently using a pink placeholder dot since the original dot in the logo camouflages with the background

rane20 committed:
Merge branch 'main' into melissa_navBar
